in-vehicle-digital-twin/src/digitaltwin_impl.rs

```diff
@@ -33,15 +34,15 @@ impl DigitalTwin for DigitalTwinImpl {
         let request_inner = request.into_inner();
         let entity_id = request_inner.entity_id;
 
-        info!("Received a find_by_id request for entity id {}", &entity_id);
+        debug!("Received a find_by_id request for entity id {entity_id}");
 
-        let lock: MutexGuard<HashMap<String, Value>> = self.entity_map.lock().unwrap();
-        let val = match lock.get(&entity_id) {
+        let lock: MutexGuard<HashMap<String, Value>> = self.entity_map.lock();
```

Comment:
Here and at other places. Consider using [RWLock](https://doc.rust-lang.org/std/sync/struct.RwLock.html). This function does a RO access and could be called in parallel with RWLock. The `register_each_one` function below would only do the write lock.

Reply:
I made this fix for the in-vehicle digital twin service.  I left it as-is for the samples.